Parser/pegen/pegen.c

```diff
@@ -2058,7 +2058,7 @@ _PyPegen_make_module(Parser *p, asdl_seq *a) {
 // Error reporting helpers
 
 expr_ty
-_PyPegen_get_invalid_target(expr_ty e, int is_del)
+_PyPegen_get_invalid_target(expr_ty e, int is_del, int is_for)
```

Comment:
Given that `is_del` and `is_for` cannot be passed together I think this should be two functions or it should receive an enum or something like that.

Reply:
Done in https://github.com/python/cpython/pull/20106/commits/c1cf8e63562999fe2ad8fac6a327e04c49f17aef.